src/rider/main/kotlin/me/fornever/avaloniarider/previewer/AvaloniaPreviewerSessionController.kt

```diff
@@ -359,4 +468,13 @@ class AvaloniaPreviewerSessionController(
     fun sendInputEventMessage(event: AvaloniaInputEventMessage) {
         session?.sendInputEventMessage(event)
     }
+
+    private fun scheduleRestart(projectFilePath: Path) {
```

Comment:
Please explain why do we need restart in certain cases. My understanding was that we usually just do live reload. Why would we need to restart?